### PR TITLE
[MLINKCHECK-36] update doxia and parent POM

### PR DESCRIPTION
**Pull request by elharo**
@hboutemy

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ under the License.
   <parent>
     <groupId>org.apache.maven.plugins</groupId>
     <artifactId>maven-plugins</artifactId>
-    <version>33</version>
+    <version>34</version>
```
Comment (hboutemy) [inline pom.xml:28]: upgrading parent is another story, and would be useful to add the project.build.outputTimestamp property

File: pom.xml [hunk 2]
```diff
@@ -62,8 +62,8 @@ under the License.
   </distributionManagement>
 
   <properties>
-    <doxiaVersion>1.4</doxiaVersion>
-    <doxia-sitetoolsVersion>1.4</doxia-sitetoolsVersion>
+    <doxiaVersion>1.9.1</doxiaVersion>
+    <doxia-sitetoolsVersion>1.9.2</doxia-sitetoolsVersion>
```
Comment (hboutemy) [inline pom.xml:66]: I did not try to run, but yes, it should be as siple as that: that's why the issue was marked as up-for-grabs

File: pom.xml (hunk 1 above)
Comment (elharo) [reply pom.xml:28]: done

File: pom.xml [hunk 3]
```diff
@@ -62,11 +62,11 @@ under the License.
...
     <mavenVersion>2.2.1</mavenVersion>
     <javaVersion>7</javaVersion>
-    <project.build.outputTimestamp>2020-04-07T21:04:00Z</project.build.outputTimestamp>
+<project.build.outputTimestamp>2020-04-04T16:07:00Z</project.build.outputTimestamp>
```
Comment (michael-o) [inline pom.xml:69]: Broken formatting

Comment (elharo) [reply pom.xml]: fixed

**Comment by jira-importer** [comment]
Resolve #52

**Comment by jira-importer** [comment]
Resolve #52

**Pull request merged**